src/components/solverComponents/InputButton.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { MouseEventHandler, ReactElement } from "react";
+import { Tooltip, Button } from "@chakra-ui/react";
+
+
+export const InputButton = (props: { icon: ReactElement, text: string, toolTipText: string, onClick?: MouseEventHandler<HTMLButtonElement> | undefined }) => (
```

Comment:
maybe extract the props type definition to an interface or make it multi-line, this is too long

Reply:
built the interface